Sources/AdminPanelProvider/Middlewares/ProtectMiddleware.swift

```diff
@@ -6,10 +6,15 @@ public typealias ProtectMiddleware = CustomUserProtectMiddleware<AdminPanelUser>
 
 /// Redirects unauthenticated requests to a supplied path.
 public final class CustomUserProtectMiddleware<U: AdminPanelUserType>: Middleware {
+    let passwordEditPathForUser: (U) -> String
```

Comment:
Do we need something default here to avoid the breaking change?

Reply:
I didn't consider that this Middleware was public. I can put in the same value as the provider gets.